src/features/game-identifier/services/game.service.js

```diff
@@ -40,11 +40,15 @@ export function discoverGamesFromSteamWeb(steamApps, htmlDetailsPages) {
 export function getReleaseDate(page) {
   const dom = new JSDOM(page);
 
-  const releaseDate = dom.window.document.querySelector(".release_date .date");
+  const releaseDateString = dom.window.document.querySelector(".release_date .date");
 
-  if (!releaseDate) return "";
+  if (!releaseDateString) return "";
 
-  return new Date(releaseDate.textContent.trim());
+  const releaseDate = new Date(releaseDateString.textContent.trim());
+
+  if (releaseDate == "Invalid Date") return releaseDateString;
+
+  return releaseDate;
 }
```

Comment:
Don't get too hung up on dates - this isn't standardized yet, there are a couple of discrepancies throughout this. I didn't want to make the PR bigger, so I opened up a new issue where I will take care of this. This happens a couple of times throughout the PR. 

https://github.com/lukatarman/steam-game-stats-backend/issues/136